packages/vuetify/src/components/VList/VListItem.sass

```
@@ -40,6 +40,15 @@
   &--selectable
     user-select: auto
 
+// Fix for IE11 where min-height does not work with
+// align-items: center in flex containers
+// https://github.com/philipwalton/flexbugs/issues/231
+.v-list-item:after
+  content: ''
+  min-height: inherit
+  font-size: 0
+
+
```

this might be cleaner  (adding to the existing v-list-item block)
```suggestion
  // Fix for IE11 where min-height does not work with
  // align-items: center in flex containers
  // https://github.com/philipwalton/flexbugs/issues/231
  &:after
    content: ''
    min-height: inherit
    font-size: 0
```

---

Thanks. I did like you suggested.